.github/workflows/go.yml

```diff
@@ -417,4 +417,4 @@ jobs:
         run: |
           git status
           git diff --exit-code
-        working-directory: source
+        working-directory: source
```

Comment:
Please keep a newline there

Reply:
Not before `working-directory`, but after it. Your editor may remove it automatically; please change its configuration to prevent that